README.md

```diff
@@ -11,37 +11,77 @@ intuitive, and predictable.
 
 ```console
 admin              Administrative commands
-alloc              allocate a new instance for interactive use
+alloc              Interactively enqueue a job within a sub allocation
```

Comment:
This isn't technically correct - we probably here just want to say `allocate a new instance` and call it a day.

Reply:
Respectfully I disagree. The flux documentation calls out the interactive nature of `flux mini alloc` multiple times and thus I think we should make that distinction here as well. Users need to know that `flux mini batch` and flux mini alloc` are counterparts. One is interactive, one is not. See references below.

> An interactive Flux instance can also be started under Flux with the [flux-mini(1)](https://flux-framework.readthedocs.io/projects/flux-core/en/latest/man1/flux-mini.html) alloc subcommand [[1]](https://flux-framework.readthedocs.io/en/latest/quickstart.html#adding-job-request-validation)

> The run and alloc commands do the same interactively, blocking until the job has completed. [[2]](https://flux-framework.readthedocs.io/projects/flux-core/en/latest/man1/flux-mini.html)

> flux-mini alloc works similarly to batch, but instead blocks until the job has started and interactively attaches to the new Flux instance (unless the --bg option is used). By default, a new shell is spawned as the initial program of the instance, but this may be overridden by supplying COMMAND on the command line. [[3]](https://flux-framework.readthedocs.io/projects/flux-core/en/latest/man1/flux-mini.html)

Reply:
The part I have issue with is saying we are "enqueuing a job." It doesn't matter if (technically speaking) this is what we are doing, the user understands this as an allocation they get. The use of interactive is fine, but the alloc needs to better speak to that case the user understands.

Reply:
So you would be okay with the following?

```
alloc              Interactively create a new allocation and flux instance
```

Reply:
Sure that works for me.